.github/dependabot.yml

```diff
@@ -1,8 +1,9 @@
----
 version: 2
 updates:
-  - package-ecosystem: "gomod"
-    directory: "/"
+  - package-ecosystem: "github-actions"
+    directories:
+      - "/"
+      - "**/*"
```

Comment:
Do we have other Github actions besides the .github directory?

Reply:
Ah, damn, my fault! For GitHub Actions `/` is fine, of course. Fixing it!